basic_samples/DataViews/Python3/README.md

```diff
@@ -29,8 +29,6 @@ or
 
 Automated test uses Python 3.6.8 x64
 
-Complete dependencies listed in [DEPENDENCIES.md](DEPENDENCIES.md)
-
 For the general steps or switch languages see the Task [ReadMe](../)  
```

Comment:
I can't put this comment on the right line but on the line 5 we don't link to the library right

Reply:
This should be fixed now